packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.ts

```diff
@@ -36,7 +39,7 @@ function getCommonQueryConditions({
   draft,
   allowAnyDraft,
 }: {
-  scope: DocumentLogWithErrorScope
+  scope: any
```

Comment:
honestly #peresa debugging the type for this local method tbh

Reply:
So all `DocumentLogWithErrorScope` was not working?

Reply:
no because now it's a dynamic query yada yada, honestly i'm pretty sure i could fix this type but since this is a private method i chose to move on with my life #sorry